types/aws-lambda/index.d.ts

```diff
@@ -36,6 +36,7 @@
 //                 Alessandro Palumbo <https://github.com/apalumbo>
 //                 Sachin Shekhar <https://github.com/SachinShekhar>
 //                 Ivan Martos <https://github.com/ivanmartos>
+//                 Zach Anthony <https://github.com/zach-anthony>
```

Comment:
Just to check you're OK with getting contacted for every PR on this package?

Reply:
Yeah, that's fine